<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagementTeacher</class>
 <widget class="QWidget" name="ManagementTeacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="groupBox_search">
     <property name="title">
      <string>查询</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="49,6">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="12,12,6,7,12">
          <property name="spacing">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,10">
            <item>
             <widget class="QLabel" name="label_search_tno">
              <property name="text">
               <string>教工号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_search_tno"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,10">
            <item>
             <widget class="QLabel" name="label_search_tname">
              <property name="text">
               <string>姓名</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_search_tname"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,4">
            <item>
             <widget class="QLabel" name="label_search_tsex">
              <property name="text">
               <string>性别</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_search_tsex">
              <item>
               <property name="text">
                <string>不限</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>男</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>女</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,2,1,2">
            <item>
             <widget class="QLabel" name="label_search_tage">
              <property name="text">
               <string>年龄</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_search_tage_a"/>
            </item>
            <item>
             <widget class="QLabel" name="label_search_tage_">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_search_tage_b"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,10">
            <item>
             <widget class="QLabel" name="label_search_cno">
              <property name="text">
               <string>主讲课</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_search_cno"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="spacing">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_search_teb">
              <property name="text">
               <string>学历</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_search_teb">
              <property name="text">
               <string>学士</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_search_teb_2">
              <property name="text">
               <string>硕士</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_search_teb_3">
              <property name="text">
               <string>博士</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_search_tpt">
              <property name="text">
               <string>职称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_search_tpt">
              <property name="text">
               <string>助教</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_search_tpt_2">
              <property name="text">
               <string>讲师</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_search_tpt_3">
              <property name="text">
               <string>副教授</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_search_tpt_4">
              <property name="text">
               <string>教授</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_search">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="6,2">
     <item>
      <widget class="QGroupBox" name="groupBox_tableView">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_management">
         <property name="title">
          <string>信息管理</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_add">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_delete">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_change">
            <property name="text">
             <string>修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_excel">
            <property name="text">
             <string>打印</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_data">
         <property name="title">
          <string>教师信息</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_tno">
              <property name="text">
               <string>教工号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_tno">
              <property name="maxLength">
               <number>8</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_tname">
              <property name="text">
               <string>姓名</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_tname">
              <property name="maxLength">
               <number>8</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_tsex">
              <property name="text">
               <string>性别</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_tsex">
              <item>
               <property name="text">
                <string>男</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>女</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_tage">
              <property name="text">
               <string>年龄</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_tage">
              <item>
               <property name="text">
                <string>24</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>25</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>26</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>27</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>28</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>29</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>31</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>33</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>34</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>35</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>36</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>37</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>38</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>39</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>40</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>41</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>42</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>43</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>44</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>45</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>46</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>47</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>48</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>49</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>50</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>51</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>52</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>53</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>54</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>55</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>56</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>57</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>58</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>59</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>60</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_teb">
              <property name="text">
               <string>学历</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_teb">
              <item>
               <property name="text">
                <string>学士</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>硕士</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>博士</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_tpt">
              <property name="text">
               <string>职称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_tpt">
              <item>
               <property name="text">
                <string>助教</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>讲师</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>副教授</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>教授</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_cno1">
              <property name="text">
               <string>主讲课程一</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_cno1">
              <property name="maxLength">
               <number>6</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="label_cno2">
              <property name="text">
               <string>主讲课程二</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_cno2">
              <property name="maxLength">
               <number>6</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLabel" name="label_cno3">
              <property name="text">
               <string>主讲课程三</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_cno3">
              <property name="maxLength">
               <number>6</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QPushButton" name="pushButton_yes">
              <property name="text">
               <string>yes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_back">
              <property name="text">
               <string>返回</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
